### ISSUE TITLE
Question: default firmware

### ISSUE THREAD

**Issue by hryamzik** [albertskog/stellaris-evalbot-fw#1]
Does this project include the default firmware that was installed on the evalbot? I'm trying to get mine back to live and display doesn't show anything, not sure if that's a HW or FW issue so flashing defaults seems to be the best starting point.

**Comment by albertskog** [comment]
No, the copy of Stellarisware that I found does not include the example code unfortunately. Did you try running the code from this project? Because it does not use the display, I was not able to make mine work yet either..

**Comment by hryamzik** [comment]
Not yet, setting up my mac. I didn't touch this for several years.


> I was not able to make mine work yet either..

That may mean I don't have hardware issues, time to find working display code.

**Comment by hryamzik** [comment]
@albertskog can you try flashing `StellarisWare/boards/ek-evalbot/simple_display/gcc/simple_display.bin`, please?

**Comment by albertskog** [comment]
Which version of Stellarisware is this, did you get it from the link in the Readme? I don't have that path.. There is a `stellarisware/examples/boards/` but it just has a `Readme.md`...

```
➜  evalbot tree -d stellarisware
stellarisware
├── docs
├── driverlib
│   ├── ccs-cm3
│   │   └── Debug
│   ├── ewarm-cm3
│   │   └── Exe
│   ├── gcc-cm3
│   └── rvmdk-cm3
├── examples
│   ├── boards
│   ├── peripherals
│   │   ├── adc
│   │   ├── can
│   │   ├── epi
│   │   ├── i2c
│   │   ├── pwm
│   │   ├── rom
│   │   ├── ssi
│   │   ├── sysctl
│   │   ├── systick
│   │   ├── timer
│   │   └── uart
│   └── project
│       ├── ccs
│       └── gcc
└── inc
```

**Comment by hryamzik** [comment]
Nah my files are dated 2013-2019 so not sure where I got this folder from. Let me attach the bin here.
[simple_display.bin.zip](https://github.com/albertskog/stellaris-evalbot-fw/files/9814105/simple_display.bin.zip)

**Comment by hryamzik** [comment]
[simple_display.zip](https://github.com/albertskog/stellaris-evalbot-fw/files/9814135/simple_display.zip)

Actually I can attach the whole project.

**Comment by albertskog** [comment]
Sorry for the delay - finally found where my 3-year-old hid the bot!
My display is not working with this bin either, only the leds are blinking.. I have not looked at the project code yet, did you find a solution?

**Comment by hryamzik** [comment]
Thanks for the update. No, I'm busy with other stuff. But that may mean that's a software issue, unless we both have the same hw malfunction.

Maybe there's another project to try: https://github.com/fcayci/LM3S9B92-EVALBOT
